src/hotspot/cpu/s390/s390.ad

```diff
@@ -6253,6 +6253,29 @@ instruct divI_reg_imm16(roddRegI dst, iRegI src1, immI16 src2, revenRegI tmp, fl
   ins_pipe(pipe_class_dummy);
 %}
 
+// Unsigned Integer Register Division
+// NOTE: z_dlr requires even-odd pair. remainder will be in even register(r4) & quotient will be stored in odd register(r5)
+// for dividend, leftmost 32bits will be in r4 and rightmost 32bits will be in r5 register.
```

Comment:
```suggestion
// for dividend, upper 32bits will be in r4 and lower 32bits will be in r5 register.
```

This PR uses the terms "leftmost" and "rightmost" for numerically higher and lower parts of an integer. I wondered if this perhaps was common in s390.ad, but it is not. Please say "higher" and "lower".

Reply:
Or even "low" and "high".

Reply:
I followed the PofZ book and took "leftmost" and "rightmost" from it. But now I have updated it to "upper" and "lower" that feels more intuitive.